app/views/shared/_email_subscribe_unsubscribe_flash.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if @account_flash.include?("email-subscription-success") %>
+
+  <%= render "govuk_publishing_components/components/success_alert", {
+    message: "You've subscribed to emails about this page.",
```

Comment:
I tried using the fancy apostrophe (`’`) here, but I got a `ActionView::Template::Error: "\xE2" from ASCII-8BIT to UTF-8
` error when trying to render the view. If there's an easy way to sort this, let me know otherwise I think the regular one is fine

Reply:
Is it possible your editor saved the file with some encoding other than utf-8?  https://stackoverflow.com/a/5286382 suggests it could happen if the encoding is wrong.

Otherwise, I think `&apos;` is the HTML entity for the apostrophe we want.

Reply:
My editor _says_ it's saving files with UTF-8. I'll try the HTML entity

Reply:
`&rsquo;` (right single quotation mark) looks like the correct entity for this:

![image](https://user-images.githubusercontent.com/6362602/140726793-69b59f2b-d271-49d7-8f8f-8a5a4547a023.png)